### PR TITLE
Add a license statement in the footer

### PR DESCRIPTION
**Pull request by michielappelman**
### Prerequisites

- [ ] This pull request fixes a bug.
- [x] This pull request adds a feature.
- [ ] This pull request introduces breaking change.

### Description

Although a copyright for the blog's content is included in the footer, it might be worthwhile to add a license to the page as well.

This PR proposes an initial way to do so, using a site parameter. It also adds initial EN and NL i18n statements referring to the license. They are shorter than what [CC actually recommends](https://creativecommons.org/choose/results-one?license_code=by-sa), but should suffice for most uses of Hugo and Coder. More i18n translations will also need to be added.

I am fairly new to the Hugo ecosystem, so there might be better ways to do this. Please feel free to discuss and provide any feedback.

### Issues Resolved

~~List any existing issues this pull request resolves.~~

#### General

This PR changes 4 files initially:
- Footer layout is extended with a reference to the `license` parameter, directly after the author's name and a i18n string referring to the license (`licensed_under`).
- The `config.toml` example file now includes a (commented out) `license` parameter
- The EN and NL i18n include the `licensed_under` keys

- [x] Describe what changes are being made
- [x] Explain why and how the changes were necessary and implemented respectively
- [x] ~~Reference issue with `#<ISSUE_NO>` if applicable~~

#### Resources

- [x] ~~If you have changed any SCSS code, run `make release` to regenerate all CSS files~~

#### Contributors

- [x] Add yourself to `CONTRIBUTORS.md` if you aren't on it already

### CODE REVIEW

**Comment by metbril** [comment]
I suggest that this PR should add the new key to all translations, where the English translation is used as an example. See https://github.com/luizdepra/hugo-coder/commit/963a78323b0fc8474c0d3b03fdf7577e03454d3b

**Comment by michielappelman** [comment]
Thanks for the feedback @metbril. The example translations have been added.

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -7,6 +7,10 @@
     {{ now.Year }}
     {{ with .Site.Params.author }} {{ . }} {{ end }}
     ·
+    {{ if (and .Site.Params.license) }}
+      {{ i18n "licensed_under" }} {{ .Site.Params.license }}
```
Comment (metbril) [inline layouts/partials/footer.html:11]: Perhaps this should be filtered with `safeHTML`:

```
{{ .Site.Params.license | safeHTML }}
```

https://gohugo.io/functions/safehtml/

Comment (michielappelman) [reply layouts/partials/footer.html:11]: That's a good one, that would allow us to include a link to the license. Will update this one.

**Comment by luizdepra** [comment]
LGTM. Ty.

**Pull request merged**